libs/input-mapping/src/File/Options/RewrittenInputFileOptions.php

```diff
@@ -77,6 +77,9 @@ public function getStorageApiFileListOptions(InputFileStateList $states): ListFi
                 // intentionally blank, no state configuration
             }
         }
+        if (isset($this->definition['file_ids']) && count($this->definition['file_ids'])) {
+            $options->setQuery('id:(' . implode(' OR ', $this->definition['file_ids']) . ')');
```

Comment:
Stahovani podle vic ID funguje takhle. Takze asi bych to v tuhle chvili nelimitoval jen na jeden soubor 🤔

Reply:
supr